code/_helpers/logging.dm

```diff
@@ -27,7 +27,7 @@
 
 /proc/log_debug(text)
 	if (config.log_debug)
-		diary << "\[[time_stamp()]]DEBUG: [text][log_end]"
+		debug_log << "\[[time_stamp()]]DEBUG: [text][log_end]"

```

Comment:
Should we be switching other debug-logging things to use this as well, if/when this is merged?

Reply:
Its a nice idea, but this PR will work fine even before any of that is done.